src/Draggable/Plugins/Mirror/README.md

```diff
@@ -25,36 +26,71 @@ Defining this sets the offset from cursor to mirror manually on the x axis. Defa
 **`cursorOffsetY {Number|null}`**  
 Defining this sets the offset from cursor to mirror manually on the y axis. Default: `null`
 
+**`appendTo {String|HTMLElement|Function}`**  
+The mirror plugin allows you to specify where the mirror should be appended to. For clarification,
+this is not where the source will be placed, only the temporary mirror element, which is the element
+that follows your cursor as you drag. You can specify a css selector, a HTMLElement or a function
+that returns a HTMLElement. Default is the source parent element.
+
+### Events
+
+| Name                                | Description                                           | Cancelable | Cancelable action        |
+| ----------------------------------- | ----------------------------------------------------- | ---------- | ------------------------ |
+| [`mirror:create`][mirrorcreate]     | Gets fired before draggable mirror gets created       | true       | Prevents mirror creation |
+| [`mirror:created`][mirrorcreated]   | Gets fired when draggable mirror gets created         | false      | -                        |
+| [`mirror:attached`][mirrorattached] | Gets fired when draggable mirror gets attached to DOM | false      | -                        |
+| [`mirror:move`][mirrormove]         | Gets fired when draggable mirror moves                | true       | Stop mirror movement     |
+| [`mirror:destroy`][mirrordestroy]   | Gets fired when draggable mirror gets removed         | true       | Stop mirror removal      |
+
+[mirrorcreate]: MirrorEvent#mirrorcreateevent
+[mirrorcreated]: MirrorEvent#mirrorcreatedevent
+[mirrorattached]: MirrorEvent#mirrorattachedevent
+[mirrormove]: MirrorEvent#mirrormoveevent
+[mirrordestroy]: MirrorEvent#mirrordestroyevent
+
 ### Examples
 
 #### y Axis only
 
 ```js
-import {Draggable} from '@shopify/draggable';
+import { Draggable } from '@shopify/draggable';
```

Comment:
interesting - did this happen automatically?

Reply:
Yup, must be some VSCode config. Have a suspicion it comes from `editorconfig`, but may be wrong. Let's investigate later